recipes/botan/all/conanfile.py

```diff
@@ -44,7 +44,7 @@ def configure(self):
         if Version(self.version) >= "2.14.0":
             self._validate_v2_14()
 
-        if self.options.single_amalgamation:
+        if self.options.get_safe("single_amalgamation"):
             self.options.amalgamation = True
```

Comment:
```suggestion
        self.options.amalgamation = self.options.get_safe("single_amalgamation", False)
```

Reply:
you don't need to apply it if you want not. keep in mind get_safe has a default value as second parameter.

Reply:
Good to know, thanks. In that case it's not the same though, if `single_amalgamation` is not set, users can still use `amalgamation=True`. It's just an implication `single_amalgamation => amalgamation`.